frontend/src/components/Error/Error.jsx

```diff
@@ -1,24 +1,24 @@
 import "./Error.css";
-import { useState, useEffect, useContext } from "react";
+import { useEffect, useContext } from "react";
 import { UserContext } from "../App/App";
+import { useLocation } from "react-router-dom";
 
 export default function Error({ error }) {
```

Comment:
If the variable `error` is a string, then a better name for it is "errorMessage", "errorMsg", or "message".